webapp/src/components/onboardingTour/shareBoard/shareBoard.tsx

```diff
@@ -30,23 +28,25 @@ const ShareBoardTourStep = (): JSX.Element => {
     )
 
     const punchout = useMeasurePunchouts(['.ShareBoardButton > button'], [])
-    const currentStep = parseInt(useAppSelector(getOnboardingTourStep), 10)
 
     return (
-        <TourTip
-            screen={screen}
-            title={title}
-            punchOut={punchout}
-            step={currentStep}
-            tutorialCategory={TOUR_CARD}
-            autoTour={true}
-            placement={'bottom-end'}
-            className='ShareBoardTourStep'
-            hideBackdrop={true}
-            imageURL={Utils.buildURL(shareBoard, true)}
-            skipCategoryFromBackdrop={true}
-            telemetryTag='tourPoint2b'
-        />
+        <>
```

Comment:
nit: I would use something like:

```js
if (!BoardTourSteps.SHARE_BOARD) {
  return null
}
```

and then return the `TourTipRenderer` thing. That way is just cleaner and simpler.

Reply:
Done.